app/livechat/lib/LivechatInquiry.js

```diff
@@ -100,6 +142,29 @@ if (Meteor.isServer) {
 
 			return this.update(query, update);
 		}
+
+		async getCurrentSortedQueue({ _id, department }) {
+			const collectionObj = this.model.rawCollection();
+			const aggregate = [
+				{
+					$match: {
+						status: 'queued',
+						...department && { department },
+					},
+				},
+				{ $sort: { ts: 1 } },
+				{ $group: { _id: 1, inquiry: { $push: { _id: '$_id', rid: '$rid', name: '$name', ts: '$ts', status: '$status', department: '$department' } } } },
```

Comment:
Can we break and indent every step as the first one to make it more readable?

Reply:
Sure!
Done.